lib/LaTeXML/Package/pstricks.sty.ltxml

```diff
@@ -547,37 +547,35 @@ DefEnvironment('{pspicture} [Float] PSCoord OptionalPSCoord',
     my ($c0,      $c1)      = ($whatsit->getArg(2), $whatsit->getArg(3));
     if (!$c1) {    # Note: the FIRST coord is optional! so if 2nd missing, swap.
       $c1 = $c0; $c0 = ZeroPair(); }
-    $whatsit->setProperty(c0 => $c0);
-    $whatsit->setProperty(c1 => ($c1 ? Pair($c1->getX->subtract($c0->getX),
-          $c1->getY->subtract($c0->getY)) : $c0));
-
-    # Seems we need to account for the origin here.
     AssignValue('\psorigin', Pair($c0->getX->negate, $c0->getY->negate));
     my $org = getNecessaryTransform();
     AssignValue('_psActiveCCTransform', $org);
     ackTransform($org);
-    $whatsit->setProperties(need => 1, transform => $org->ptValue);
-
+    $whatsit->setProperties(need => 1, transform => $org->ptValue,
```

Comment:
Just checking: the transform here is still in `ptValue` even if all others are switching to px?

Reply:
Possibly; transform in this case is a simple dimension, not the css/svg one. Lack of testing.